src/error/parser.cpp

```diff
@@ -52,6 +52,7 @@ parse_t parse (std::string const & file)
 			bool isHighlevelFile = (file.find ("highlevel") != string::npos);
 			if (!std::regex_match (text, codeRegex) && !isHighlevelFile)
 				throw parse_error ("Error code does not match regular expression [0-9A-Z]{5,5}", linenr);
+			text = "C"+text;
```

Comment:
@markus2330 is it ok to deal it in that way or should we enforce it in the specification file for consistency (wysiwyg)

Reply:
I do not know what you try to make here. Yes, in general consistency should be enforced.

Reply:
Either we force the user to write in the specification file `01000` and the parser prepends the `C` or we force the user to write `C01000` into the specification file. We agrees in the meeting that error codes should have a prepended `C`.

Reply:
I see. Yes, better let the user write the complete code in the specification and enforce the C in the beginning.